contexts/SupabaseContext.tsx

```diff
@@ -179,14 +148,14 @@ function SupabaseProvider({
           throw new Error("No user data found");
         }
 
-        const { username: username_, ranks, onboarding } = data;
+        const { username: username_, ranks, bio } = data;
         const fetchedRank = ranks.length > 0 ? ranks[0].rank : null;
-        const onboardStatus =
-          onboarding.length > 0 ? onboarding[0].status : null;
+
+        setProfileComplete(bio !== null);
```

Comment:
I know on the frontend we don't allow for `bio` to be submitted as an empty string, but in the case that happens, do we want to consider a profile complete even if `bio === ""`? Alternatively, we could make this a truthy comparison as `setProfileComplete(bio)` (or `!!bio`)

Reply:
That's a good question — don't think we need to worry too much about it, `bio === ""` would be an invalid state (since it starts as `null` and then must be set to a non-empty string) and I don't think we need to worry about handling all invalid states (that would be a lot of work :joy:)